src/Components/ArticlePreviewSection/ArticlePreviewSection.tsx

```diff
@@ -3,57 +3,18 @@
 import ArticleCard from "@/Components/ArticleCard/ArticleCard";
 import styles from "@/Components/ArticlePreviewSection/ArticlePreviewSection.module.scss";
 import { useUserContext } from "@/context/UserPrefsContext";
+import { PostType } from "@/lib/types/postsType";
 
-export type ArticleCardPropsType = {
-  featuredImage: {
-    node: {
-      altText: string;
-      mediaItemUrl: string;
-      mediaDetails: {
-        sizes: Array<{
-          sourceUrl: string;
-          height: number;
-          width: number;
-        }>;
-      };
-    };
-  };
-  id: string;
-  title: string;
-  excerpt: string;
-  content: string;
-  slug: string;
-  tags: {
-    nodes: Array<{ name: string }>;
-  };
-  author: {
-    node: {
-      firstName: string;
-      lastName: string;
-      slug: string;
-    };
-  };
-  date: string;
-  modified: string;
-  categories: {
-    nodes: Array<{ name: string }>;
-  };
-};
-
-const ArticlePreviewSection = ({
-  articlesData,
-}: {
-  articlesData: ArticleCardPropsType[];
-}) => {
+const ArticlePreviewSection = ({ posts }: { posts: PostType[] }) => {
   const { layoutStyle } = useUserContext();
 
   return (
     <div className={styles.articleContainer}>
       <ul
         className={`${layoutStyle === "grid" ? styles.articleGrid : styles.articleList}`}
       >
-        {articlesData ? (
-          articlesData.map((articleData: ArticleCardPropsType) => (
+        {posts ? (
```

Comment:
What is this condition used for?

Reply:
I'm just checking if the articles exist then render them. I'll change the confusing names.

Reply:
Fixed